apps/meteor/server/database/watchCollections.ts

```diff
@@ -34,6 +34,8 @@ export function getWatchCollections(): string[] {
 		LivechatInquiry.getCollectionName(),
 		LivechatDepartmentAgents.getCollectionName(),
 		LoginServiceConfiguration.getCollectionName(),
+		Roles.getCollectionName(),
+		Rooms.getCollectionName(),
```

Comment:
I guess these are back due to conflicts maybe? you forgot to remove `LoginServiceConfiguration` from the list though
```suggestion
```